src/scenes/home/informationForm/informationForm.js

```diff
@@ -58,6 +59,18 @@ class SignupInformation extends Component {
     this.setState({ interests });
   }
 
+  // make sure isDescriptionValid is false on initialization
+  onIdentifierInit = () => {
```

Comment:
Are we able to do `onIdentifierInit() {`? What drove the decision to use the arrow function outside of ES6? You _should™_ have scope to `this` here.

Reply:
I don't think there is a problem to write the method the traditional way.
I chose to use the arrow function because it seems that most functions are written that way in the project.